docs/topics/health-check-mission-resources.adoc

```diff
@@ -7,3 +7,15 @@ More background and related information on health checking can be found here:
 * link:https://kubernetes.io/docs/user-guide/walkthrough/k8s201/#health-checking[Health Checking in Kubernetes]
 * link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[Kubernetes Liveness and Readiness Probes]
 * link:https://kubernetes.io/docs/api-reference/v1/definitions/#_v1_probe[Kubernetes Probes]
+
+ifndef::http-api-spring-boot-tomcat[]
+* link:{link-mission-health-check-spring-boot-tomcat}[Health Check - {SpringBoot} Booster]
+endif::http-api-spring-boot-tomcat[]
+
+ifndef::http-api-vertx[]
+* link:{link-mission-health-check-vertx}[Health Check - {VertX} Booster]
+endif::http-api-vertx[]
+
+ifndef::http-api-wf-swarm[]
+* link:{link-mission-health-check-wf-swarm}[Health Check - {WildFlySwarm} Booster]
+endif::http-api-wf-swarm[]
```

Comment:
The `link:` directives look OK, but the `ifndef:` ones all refer to `http-api`, which seems wrong?